tests/tests/hybrid/ecies_aead_hkdf_dem_helper_test.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright 2021 The Tink-Rust Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+use lazy_static::lazy_static;
+use tink_core::subtle::random::get_random_bytes;
+use tink_hybrid::subtle::EciesAeadHkdfDemHelper;
+use tink_tests::proto_encode;
+
+lazy_static! {
```

Comment:
FYI OnceCell seems to be most popular option these days: https://users.rust-lang.org/t/lazy-static-vs-once-cell-oncecell/58578 .

Reply:
Thanks for the pointer.  (I'll stick with `lazy_static` here though, as it's consistent with all the other crates and doesn't pull in a new dependency.)